cmake/multilib.yaml.in

```diff
@@ -138,3 +138,8 @@ Mappings:
 - Match: -march=thumbv8\.[1-9]m\.main(\+[^\+]+)*\+lob(\+[^\+]+)*
   Flags:
   - -march=thumbv8.1m.main+lob
+
+# -mbranch-protection options
+- Match: -mbranch-protection=(standard|pac-ret(\+leaf)?\+bti|bti\+pac-ret(\+leaf)?)
+  Flags:
+  - -mbranch-protection=pac-ret+bti
```

Comment:
I believe this is wrong. Since the multilib flag is named `-mbranch-protection=standard`, this match and insertion of `-mbranch-protection=pac-ret+bti` doesn't do anything.

Furthermore, the PACBTI library variant should also match when only one of BTI or PAC-RET is requested. Therefore the regex should also accept `pac-ret(+leaf)?` or `bti`.